modules/manage/pages/schema-reg/schema-reg-authorization.adoc

```diff
@@ -0,0 +1,355 @@
+= Schema Registry Authorization
+:page-categories: Management, Schema Registry, rpk, Security
+:page-aliases: manage:schema-reg-authorization.adoc
+:description: Learn how to set up and manage Schema Registry Authorization using ACL definitions that control user access to specific Schema Registry operations.
+// tag::single-source[]
+
+You can use Schema Registry Authorization to control user access to specific Redpanda Schema Registry resources through Access Control Lists (ACLs).
+
+ifndef::env-cloud[]
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+endif::[]
+
+== About Schema Registry Authorization
+
+Schema Registry Authorization allows you to control which users and applications can perform specific operations within the Redpanda Schema Registry. This ensures that only authorized entities can read, write, modify, or delete schemas.
+
+=== What changed in release 25.2
+
+Before release 25.2, Schema Registry supported authentication, but once a user was authenticated, they had full access to all Schema Registry operations, including reading, modifying, and deleting schemas and configuration both per-subject and globally.
+
+Starting in release 25.2, Schema Registry Authorization provides fine-grained access control through ACLs. You can now restrict access to specific subjects and operations.
+
+=== How to manage Schema Registry Authorization
+
+You can manage Schema Registry Authorization in two ways:
+
+- **Using rpk**: Use the xref:reference:rpk/rpk-security/rpk-security-acl-create.adoc[`rpk security acl create`] command, just like you would for other Kafka ACLs.
+- **Using the API**: Use the xref:api:ROOT:schema-registry-api.adoc#get-/security/acls[Redpanda Schema Registry API] endpoints.
+
+=== Schema Registry ACL resource types
+
+Schema Registry Authorization introduces two new ACL resource types in addition to the standard Kafka ACL resources (`topic`, `group`, `cluster`, and `transactionalID`):
+
+* **`registry-global`**: Controls access to global, or top-level Schema Registry operations.
+* **`registry-subject`**: Controls access for specific Schema Registry subjects.
```

Comment:
```suggestion
* **`registry`**: Controls access to global, or top-level Schema Registry operations.
* **`subject`**: Controls access for specific Schema Registry subjects.
```

registry-global and registry-subject are rpk-only flags, but in the table responses, the user will see `REGISTRY` or `SUBJECT`, and in the future, this will be the same wording used in Console

Reply:
@BenPope @r-vasquez Not sure which presentation to use--what's shown in the output/REST API, or what users must type in as part of the rpk flags. 

I just downloaded the latest, and it still shows the options as:
```
--registry-global                Whether to grant ACLs for the schema registry
--registry-subject strings       Schema Registry subjects to grant ACLs for (repeatable) 
```

For now, updated to include everything here, but in the end, it says it is listing "new resource types", so we really should use the _actual resource type name_. Please advise.